src/main/kotlin/no/nav/helse/spion/vedtaksmelding/VedtaksmeldingErrorHandling.kt

```diff
@@ -14,3 +15,46 @@ interface FailedVedtaksmeldingRepository {
     fun delete(id: UUID)
 }
 
+class PostgresFailedVedtaksmeldingRepository(val dataSource: DataSource) : FailedVedtaksmeldingRepository {
+    private val tableName = "failedvedaksmelding"
+
+    private val insertStatement = "INSERT INTO $tableName(messageData, errorMessage, id) VALUES(?::json, ?, ?::uuid)"
```

Comment:
Skal vi ikke bruke noe API for database? Dette blir ganske low level.

Reply:
Low level <3